esmvaltool/cmorizers/data/formatters/datasets/esacci_sst.ncl

```diff
@@ -0,0 +1,177 @@
+; #############################################################################
```

Comment:
I think there is an issue with this cmorizer which was probably updated after this PR. The current [script ](https://github.com/ESMValGroup/ESMValTool/blob/main/esmvaltool/cmorizers/obs/cmorize_obs_esacci_sst.py) is in Python and deals with another version of the data than this NCL script.

Reply:
Removed!